app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -1045,6 +1077,17 @@ class PipelineSampleReport extends React.Component {
                                 </li>
                               )
                             })}
+                              <br /><div className="divider" /><br />
+                              <li key='phage'>
+                                <input type="checkbox"
+                                className="filled-in cat-filter"
+                                id='phage'
+                                value='phage'
+                                onChange={(e) => {}}
```

Comment:
I think you can remove this empty onChange function.

Reply:
Turns out it's a way to avoid the following warning:
```
Warning: Failed prop type: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`
```

Reply:
Using `defaultChecked` doesn't give the desired behavior.